pkg/config/config.go

```diff
@@ -1056,6 +1062,13 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		return err
 	}
 
+	if c.TimeZone != "" && !strings.EqualFold(c.TimeZone, "local") {
+		_, err := time.LoadLocation(c.TimeZone)
+		if err != nil {
+			return fmt.Errorf("invalid timezone %s", c.TimeZone)
```

Comment:
```suggestion
			return fmt.Errorf("invalid time zone: %s", c.TimeZone)
```

Reply:
However, here I am on the fence about whether to reference the configuration file item or spell it as "time zone" (separate words). Yes, it might be a nit pick. :smile:

Reply:
Why? For consistency, the command-line option uses a dash to separate words, the English language seems to favour "time zone" as separate words, and then we have `time_zone` in the configuration file.